src/tokens-studio/spectrum2-colors/$themes.json

```diff
@@ -1335,8 +1338,7 @@
       "Palette.transparent-white.700": "fbac60c7cd862f46eba4908ae9cc5c7346156320",
       "Palette.transparent-white.800": "ea5e834dbe202dc1dbbce86ba074ab50155812c1",
       "Palette.transparent-white.900": "cc73dfce811bf9bab293a95d9342fd6a9135c9ab",
-      "Palette.transparent-white.1000": "0ae4fb367bd6d70103b8df1fd1fff8a237a94b30",
-      "Icon.primary.disabled.default": "99b973292918e2c767311e3bca5831040c7f9ae9"
+      "Palette.transparent-white.1000": "0ae4fb367bd6d70103b8df1fd1fff8a237a94b30"
     },
```

Comment:
@nabuhasan not sure, why the changes above appear here. Again, we are not in control about how FTS is syncing tokens and variables, but these changes are all okay. It is just the order of the entries, the "token name to variable id" mapping is correct. But thanks for the extra eyes. This data is really "sensitive" and we should be extra careful with these entries. 👍

Reply:
Thanks for that clarification @karstens!